packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -665,6 +669,7 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
           CpuProfileData.nameKey: verboseName,
           CpuProfileData.categoryKey: category,
           CpuProfileData.resolvedUrlKey: rawUrl,
+          CpuProfileData.processedUrlKey: processedUrl,
```

Comment:
could you add sourceLine here too

Reply:
Going to handle this in https://github.com/flutter/devtools/issues/3974